sway-book/src/documentation/language/built-ins/numeric.md

```diff
@@ -0,0 +1,42 @@
+# Numeric Types
+
+Broadly speaking there are two types of integers:
+
+<!-- no toc -->
+- [Signed](#signed-integers) (positive and negative)
+- [Unsigned](#unsigned-integers) (only positive)
+
+## Signed Integers
+
+A signed integer is a whole number which can take the value of zero and both negative and positive values. This means that a signed integer can take values such as:
+
+- -42
+- 0
+- 42
+
+In order to achieve this one _bit_ must be kept for tracking the sign (+ or -) of the value and thus the range of available values is smaller than an unsigned integer.
+
+For those inclined, the range for an n-bit signed integer is -2<sup>n-1</sup> to 2<sup>n-1</sup>-1.
+
+Sway does not natively support signed integers however there is nothing stopping a library from using primitives to create types that act like signed types.
```

Comment:
Should we link to the existing library in sway-libs? On the one hand, if someone is looking for signed int we should point them in the right direction. On the other hand, we probably don't want the book to depend on types that are maintained in a different repo.